src/main/scala/scalismo/faces/color/HSV.scala

```diff
@@ -19,34 +19,34 @@ package scalismo.faces.color
 /** HSV color value (Hue, Saturation, Value) */
 case class HSV(hue: Double, saturation: Double, value: Double) {
 
-  /** convert to RGB value */
-  def toRGB: RGB = {
+  /** convert to sRGB value */
+  def tosRGB: sRGB = {
```

Comment:
Yes it is, HSV depends on an underlying color space. Most of the times when someone uses HSV it is on sRGB. Therefore, we decided to convert between HSV and sRGB.